osschallenge/templates/osschallenge/projectindex.html

```diff
@@ -26,7 +26,7 @@
     </div>
     {% if user.profile.role.id == mentor_id %}
         <a href="new_project/">
-            <button class="btn btn-primary">{% trans "New Project" %}</button>
+            <button id="new-project" class="btn btn-primary">{% trans "New Project" %}</button>
```

Comment:
Can you refactor this to only have the "New project" button once in the template?

Reply:
I changed it.

Reply:
👍